Patch by Sander Hoentjen <sander@hoentjen.eu> for zarafa >= 7.1.4, which ensures
building with swig 2.0.x by using PyInt_FromLong instead of SWIG_From_long.

--- zarafa-7.1.4/swig/icalmapi.i		2013-02-28 16:13:24.000000000 +0100
+++ zarafa-7.1.4/swig/icalmapi.i.swig20		2013-03-23 23:22:08.000000000 +0100
@@ -24,14 +24,14 @@
 	$1 = &temp;
 }
 %typemap(argout) (eIcalType* ) {
-	%append_output(SWIG_From_long(*$1));
+	%append_output(PyInt_FromLong(*$1));
 }
 
 %typemap(in,numinputs=0) (time_t *) (time_t temp) {
 	$1 = &temp;
 }
 %typemap(argout) (time_t* ) {
-	%append_output(SWIG_From_long(*$1));
+	%append_output(PyInt_FromLong(*$1));
 }
 
 %typemap(in,numinputs=0) (SBinary *) (SBinary temp) {
